cypress/integration/table.cy.js

```diff
@@ -153,77 +206,43 @@ const assertDimensions = (inputDimensions) => {
     }
 }
 
-describe('table', () => {
-    beforeEach(() => {
-        cy.visit('/', EXTENDED_TIMEOUT)
-
-        // remove org unit
-        cy.getBySel('layout-chip-ou').findBySel('dimension-menu-button').click()
-        cy.containsExact('Remove').click()
-    })
-    it('click on column header opens the dimension dialog', () => {
-        const dimensionName = TEST_DIM_TEXT
-
-        selectEventProgramDimensions({
-            ...event,
-            dimensions: [dimensionName],
-        })
-
-        const testDimensions = mainAndTimeDimensions.map(
-            (dimension) => dimension.label
-        )
+const init = () => {
+    cy.visit('/', EXTENDED_TIMEOUT)
 
-        // add main and time dimensions
-        testDimensions.forEach((label) => {
-            cy.getBySel('main-sidebar')
-                .contains(label)
-                .closest(`[data-test*="dimension-item"]`)
-                .findBySel('dimension-menu-button')
-                .invoke('attr', 'style', 'visibility: initial')
-                .click()
+    // remove org unit
+    cy.getBySel('layout-chip-ou').findBySel('dimension-menu-button').click()
+    cy.containsExact('Remove').click()
+}
 
-            cy.contains('Add to Columns').click()
+describe(['<40'], 'table', () => {
+    beforeEach(init)
+    it('click on column header opens the dimension dialog', () => {
+        programDimensions.push({
+            label: 'Analytics - Number (option set)',
+            value: 1,
         })
+        assertColumnHeaders()
+    })
+    it('dimensions display correct values in the visualization', () => {
+        assertDimensions()
+    })
+})
 
-        selectFixedPeriod({
-            label: periodLabel,
-            period: {
-                type: 'Daily',
-                year: `${getPreviousYearStr()}`,
-                name: `${getPreviousYearStr()}-12-10`,
-            },
+describe(['dev', '>=40'], 'table', () => {
```

Comment:
The 'dev' tag can be removed for now. If we change the way we do tags, we can always add it in later. Also, could you move the comment on line 232 down to line 239 since it is only relevant to the program dimension, not scheduled date?